service/resource/configmap/desired_test.go

```diff
@@ -749,6 +724,86 @@ func Test_Resource_ConfigMap_GetDesiredState(t *testing.T) {
 					ScrapeTimeout:      model.Duration(10 * time.Second),
 					EvaluationInterval: model.Duration(1 * time.Minute),
 				},
+				ScrapeConfigs: []*config.ScrapeConfig{
```

Comment:
Git chokes here weirdly. I don't get it. 

tl;dr - removed the label from this test, added the test beneath. The diff magic makes no sense to me sometime.